source/predicates/predicate.ts

```diff
@@ -237,7 +237,7 @@ export class Predicate<T = unknown> implements BasePredicate<T> {
 
 	@param validator - Validator to register.
 	*/
-	protected addValidator(validator: Validator<T>) {
```

Comment:
Why are you removing `protected`?

Reply:
Because `addValidator`is used in `operators/not.ts`.
https://github.com/sindresorhus/ow/blob/7ca2f75a197e3678cf0688bd3122d1d88827f5a2/source/operators/not.ts#L12-L14

`not` function isn't in subclass, so if not removing `protected`, following error occured.
`Property 'addValidator' is protected and only accessible within class 'Predicate<T>' and its subclasses.ts(2445)`
<img width="1014" alt="スクリーンショット 2021-01-27 8 23 38" src="https://user-images.githubusercontent.com/50798936/105919431-fce2dc00-6078-11eb-8233-27c2cd7ff2f1.png">